app/src/constants/paths.js

```diff
@@ -2,3 +2,4 @@ import {normalize} from 'path'
 
 export const lib = normalize(`${__dirname}/..`)
 export const root = normalize(`${lib}/../..`)
+export const chat = 'chat'
```

Comment:
yeah I wasn't sure where to put this, so it goes on `app/src/constants/pages.js` ?